src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderNewTemplate.java

```diff
@@ -57,6 +57,7 @@
     public final String bindPorts;
     public final boolean bindAllPorts;
     public final boolean privileged;
+    public final boolean togglemode;
```

Comment:
exclusiveMode is better, because it's called mode in Node.Mode JavaDoc: http://javadoc.jenkins-ci.org/hudson/model/Node.Mode.html